posix.mak

```diff
@@ -496,13 +523,18 @@ docs.json : ${DMD_REL} ${DRUNTIME_STABLE_DIR} \
 	  --only-documented $(MOD_EXCLUDES_PRERELEASE)
 	rm .release-files.txt .release-dummy.html
 
-docs-prerelease.json : ${DMD} ${DRUNTIME_DIR} \
+docs-prerelease.json : ${DMD} ${DMD_DIR} ${DRUNTIME_DIR} \
 		${PHOBOS_DIR_GENERATED} | dpl-docs
+	find ${DMD_DIR}/src -name '*.d' | \
+		sed -e /objc.d/d -e /mscoff/d -e '/dmd\/src\/id.d/d' \
```

Comment:
The exclude of `src/id.d` was only necessary because I didn't completely clean the `dmd` cache. As I assume I won't be the only one, I added it to the "ignore list".